ch04/README.md

```diff
@@ -358,10 +358,10 @@ i *= static_cast<int>(d);
 >Rewrite each of the following old-style casts to use a named cast:
 ```cpp
 int i; double d; const string *ps; char *pc; void *pv;
-pv = (void*)ps; // pv = const_cast<string*>(ps); or pv = static_cast<void*>(const_cast<string*>(ps));
+pv = (void*)ps; // pc = reinterpret_cast<void*>(ps);
```

Comment:
I was wrong for (a), it can't complied using VS2015 compiler either, I have't test it.
For (d), I just quoted 
>a pointer to any nonconst type can be converted to void* 

from C++ primer, but what I mean is  `void*` can also be converted to any nonconst type pointer `T*`(an answer on stackoverflow said that it is defined in standard, but i haven't find it), so using `static_cast` is enough. 
Some references I've found:
[Should I use static_cast or reinterpret_cast when casting a void* to whatever
](http://stackoverflow.com/questions/310451/should-i-use-static-cast-or-reinterpret-cast-when-casting-a-void-to-whatever)
[Why must I use a cast to convert from void*?](http://www.stroustrup.com/bs_faq2.html#void-ptr) by *Stroustrup*